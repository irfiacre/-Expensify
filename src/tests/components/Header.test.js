import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';


// Resource: https://enzymejs.github.io/enzyme/docs/api/shallow.html
// Jest (toJSON config): https://jestjs.io/docs/en/configuration.html

describe('*************** Testing the Header component ***************',()=>{
    test('Should render the Header correctly', ()=>{
        const wrapper = shallow( <Header /> );
        expect(wrapper).toMatchSnapshot();
    });

});
