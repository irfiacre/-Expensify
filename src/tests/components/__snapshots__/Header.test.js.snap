// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`*************** Testing the Header component *************** Should render the Header correctly 1`] = `
<header
  className="header"
>
  <h1>
    Expensify
  </h1>
  <div
    className="links"
  >
    <NavLink
      activeClassName="isActive"
      className="navLink"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
    <NavLink
      activeClassName="isActive"
      className="navLink"
      to="/create"
    >
      Create Expense
    </NavLink>
    <NavLink
      activeClassName="isActive"
      className="navLink"
      to="/help"
    >
      Help
    </NavLink>
  </div>
</header>
`;
